doc/source/performance/procfs-plugin.rst

```diff
@@ -0,0 +1,83 @@
+.. procfs-plugin:
+
+======================================================================
+Using the ProcFS plugin
+======================================================================
+
+
+Implemented in |Percona Server| 8.0.25-15, the ProcFS plugin provides access to the Linux performance counters by running SQL queries against a Percona Server for MySQL 8.0 (or other MySQL 8.0-compatible variant). This plugin is useful for Cloud and MySQL-as-a-service installations.
+
+The plugin reads files from /proc and /sys file systems and returns the file name and any content in rows in the `INFORMATION_SCHEMA.PROCFS` view. The schema definition of this view is:
+
+.. sourcecode:: mysql
+
+    CREATE TEMPORARY TABLE `PROCFS` (
+    `FILE` varchar(1024) NOT NULL DEFAULT '',
+    `CONTENTS` longtext NOT NULL
+    ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+ 
+
+Access to /proc and /sys files and directories is enabled through the system variable `procfs_files_spec`. This variable is global and read only and is set by using either the `mysqld` command line or by editing the :file:`my.cnf`. 
+
+The variable cannot be changed at run time to prevent a compromised account from giving itself greater access to the /proc or /sys file systems. 
+
+Any files that are not included within the `procfs_files_spec` are ignored and considered not to exist. You can write file access patterns in `glob(7) style <https://man7.org/linux/man-pages/man7/glob.7.html>`__:  /sys/block/sd[a-z]/stat;/proc/version*
+
+The default value for `procfs_files_spec` is: /proc/cpuinfo;/proc/irq//;/proc/loadavg/proc/net/dev;/proc/net/sockstat;/proc/net/sockstat_rhe4;/proc/net/tcpstat;/proc/self/net/netstat;/proc/self/stat;/proc/self/io;/proc/self/numa_maps/proc/softirqs;/proc/spl/kstat/zfs/arcstats;/proc/stat;/proc/sys/fs/file-nr;/proc/version;/proc/vmstat
+
+.. rubric:: Install plugin
+
+To install this plugin, copy the procfs.so file to the mysql plugin installation directory and execute 
```

Comment:
Does this apply to packages as well? In that case where should we get the file `procfs.so` file from?

Reply:
This is the same type of information for each plugin.

Reply:
updated.